prediction_market_agent/agents/think_thoroughly_agent/prompts.py

```diff
@@ -93,8 +93,10 @@
 You should determine the probability of the SCENARIO SCENARIO_TO_ASSESS being true, 
 considering the probabilities of the other related SCENARIOs.
 
+SCENARIOS_WITH_PROBABILITIES: 
+{scenarios_with_probabilities}
+
 SCENARIO_TO_ASSESS: {scenario_to_assess}
-SCENARIOS_WITH_PROBABILITIES: {scenarios_with_probabilities}
```

Comment:
Idk if this actually helps, but I read somewhere some time ago that gpt tends to be biased towards end of the prompt. So moved the main question to the end.